rct229/rulesets/ashrae9012019/section19/section19rule10.py

```diff
@@ -0,0 +1,271 @@
+
+    def __init__(self):
+        super(Section19Rule10, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, True),
+            required_fields={
+                "$": ["weather", "ruleset_model_instances"],
+                "weather": ["climate_zone"],
+            },
+            each_rule=Section19Rule10.RulesetModelInstanceRule(),
+            index_rmr="baseline",
+            id="19-10",
+            description="Air economizers shall be included in baseline HVAC Systems 3 through 8, and 11, 12, and 13 based on climate as specified in Section G3.1.2.6 with exceptions."
+            "1. Systems that include gas-phase air cleaning to meet the requirements of Standard 62.1, Section 6.1.2. This exception shall be used only if the system in the proposed design does not match the building design."
+            "2. Where the use of outdoor air for cooling will affect supermarket open refrigerated case-work systems. This exception shall only be used if the system in the proposed design does not use an economizer. If the exception is used, an economizer shall not be included in the baseline building design."
+            "3. Systems that serve computer rooms complying with Section G3.1.2.6.1.",
+            ruleset_section_title="HVAC - General",
+            standard_section="Section G3.1.2.6 including exceptions 1-3",
+            is_primary_rule=True,
+            list_path="ruleset_model_instances[0]",
+            data_items={"climate_zone": ("baseline", "weather/climate_zone")},
+        )
+
+    def is_applicable(self, context, data=None):
+        rmr_b = context.baseline
+        rmi_b = rmr_b["ruleset_model_instances"][0]
+
+        baseline_system_types_dict_b = get_baseline_system_types(rmi_b)
+
+        return any(
+            [
+                baseline_system_type_compare(system_type, applicable_sys_type, False)
+                for system_type in baseline_system_types_dict_b
+                for applicable_sys_type in APPLICABLE_SYS_TYPES
+            ]
+        )
+
+    class RulesetModelInstanceRule(RuleDefinitionListIndexedBase):
+        def __init__(self):
+            super(Section19Rule10.RulesetModelInstanceRule, self,).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, True),
+                each_rule=Section19Rule10.RulesetModelInstanceRule.HVACRule(),
+                index_rmr="baseline",
+                list_path="$.buildings[*].building_segments[*].heating_ventilating_air_conditioning_systems[*]",
+            )
+
+        def is_applicable(self, context, data=None):
+            climate_zone = data["climate_zone"]
+
+            return climate_zone not in NOT_APPLICABLE_CLIMATE_ZONE
+
+        def create_data(self, context, data):
+            rmi_b = context.baseline
+            rmi_p = context.proposed
+
+            hvac_system_exception_2_list = []
+            if find_all("$.refrigerated_casess", rmi_b):
```

Comment:
Spelling error:
refrigerated_cases,
schema has the same error so we should update the schema too.

The `refrigerated_cases` is in building level, so this would be: `"$.buildings[*].refrigerated_cases[*]"`

Reply:
Addressed.